Common/Tools/trackTuner.h

```diff
@@ -0,0 +1,679 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file trackTuner.h
+/// \brief Service to "smear" track parameters in MC simulations
```

Comment:
This is not a service in the O2 sense. It is rather a helper class and it might be more transparent to implement it as such and name the file `TrackTuner.h` accordingly.